doc/WorldBuilding.md

```diff
@@ -78,6 +78,16 @@ You will also need to apply your customised `TileSet.tres` to all other levels i
 
 3. Use the Inspector to customise the playback options or to use a different audio file.
 
+## Enemies & Player (Advanced)
```

Comment:
Let's save this customisation step til next week, please.
Still pending aligning our intended approach with the aims of the game and learning programs.